.gitignore

```diff
@@ -22,4 +22,5 @@ sketch
 NOTES.md
 app/.env
 app/config/*.json
-app/cypress/screenshots
\ No newline at end of file
+app/cypress/screenshots
+/.idea
```

Comment:
Check that path to .idea is relative path and not absolute (start with ./ or just .idea most likely